Source: Written by Esben Haabendal <esben@haabendal.dk>
Upstream: No
Reason: Fix OE-lite toolchain integration and mingw .exe filename extensions

--- lua-5.2.1/src/Makefile~orig	2012-03-09 17:32:16.000000000 +0100
+++ lua-5.2.1/src/Makefile	2015-04-17 14:25:12.383473503 +0200
@@ -6,13 +6,12 @@
 # Your platform. See PLATS for possible values.
 PLAT= none
 
-CC= gcc
+CC= $(CC)
 CFLAGS= -O2 -Wall -DLUA_COMPAT_ALL $(SYSCFLAGS) $(MYCFLAGS)
 LDFLAGS= $(SYSLDFLAGS) $(MYLDFLAGS)
 LIBS= -lm $(SYSLIBS) $(MYLIBS)
 
-AR= ar rcu
-RANLIB= ranlib
+AR= $(AR) rcu
 RM= rm -f
 
 SYSCFLAGS=
--- lua-5.2.1/Makefile~orig	2015-04-17 14:38:37.756781926 +0200
+++ lua-5.2.1/Makefile	2015-04-17 14:38:52.382182196 +0200
@@ -39,7 +39,7 @@
 PLATS= aix ansi bsd freebsd generic linux macosx mingw posix solaris
 
 # What to install.
-TO_BIN= lua luac
+TO_BIN= lua$(EXEEXT) luac$(EXEEXT)
 TO_INC= lua.h luaconf.h lualib.h lauxlib.h lua.hpp
 TO_LIB= liblua.a
 TO_MAN= lua.1 luac.1
